pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork repository cloned from another user (creates a remote named after your GitHub username):
+
+`hub fork`
+
+- Create a pull request from your fork:
+
+`git push {{github_username}} && hub pull-request`
```

Comment:
> does github_username here mean that the previous command created a branch named after your GH username? 

Not a branch, but a remote name. `git push origin` would push to the original repo, while `git push ericbn` would push to my fork, in my case. Maybe we can better say:

    git push {{github_username}} {{current_branch}}

`hub pull-request` asks for the pull request description, and you can manually state the base and head with `hub pull-request -b {{base}} -h {{head}}` but I think that's even more complex to explain. It has good defaults, so if I pushed the current branch to my fork before, it would assume the pull request is `ericbn:current_branch -> tldr:master`...